openmaps_auth/templates/index.html

```diff
@@ -19,14 +19,26 @@ <h3 class="mr2">Please log in to continue</h3>
 {% endif %}
 </div>
 <div class="title_text">
-    <h1 class="center_text">Welcome to {{ openmaps_auth.title }}!</h1> 
+    <h1 class="center_text">{% block title_text %}Welcome to {{ openmaps_auth.title }}!{% endblock title_text %}</h1>
 </div>
 {% if request.user.is_authenticated %}
 <div class="app_text">
-    <h1 id="default_title" data-name="FEATURED APPLICATIONS">FEATURED APPLICATIONS</h1>
+    {% block app_heading %}<h1 id="default_title" data-name="FEATURED APPLICATIONS">FEATURED APPLICATIONS</h1>{% endblock app_heading %}
     <p id="featured_application_description" class="hover_text">
     </p>
 </div>
+{% block messages %}
+{% if messages %}
+<div class="center_section">
+  <ul class="messagelist">
+    {% for message in messages %}
+      <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message|capfirst }}</li>
```

Comment:
This file doesn't need to be `.prettierignore`d if the `SyntaxError` is resolved. Something like this would work as expected:
```suggestion
      <li {% if message.tags %}class="{{ message.tags }}"{% endif %}>{{ message|capfirst }}</li>
```

Reply:
Good suggestion, fixed in 8540c18.